### PR TITLE
[pt-br] Add /docs/reference/glossary/cluster-operator.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translation for portuguese of the glossary term cluster-operator.

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/cluster-operator.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- user-type
+---
+  Uma pessoa que configura, controla e monitora clusters.
+
+<!--more--> 
+
+Sua principal responsabilidade é manter um cluster funcionando, o que pode envolver atividades periódicas de manutenção ou atualizações.<br>
+
+{{< note >}}
+Os operadores de cluster são diferentes do [padrão Operador](https://www.redhat.com/pt-br/technologies/cloud-computing/openshift/what-are-openshift-operators) que estende a API Kubernetes.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/cluster-operator.md:20]: Thanks for picking up the translation in Portuguese for the original link! 🎉

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm

File: content/pt-br/docs/reference/glossary/cluster-operator.md (hunk 1 above)
Comment (devlware) [reply content/pt-br/docs/reference/glossary/cluster-operator.md:20]: +1

**Pull request merged**